2019Labs/CustomSecurityContent/documentation/lab3_profiles.adoc

```diff
@@ -43,25 +45,36 @@ image::0-01-setup.png[]
 
 == Creating a new empty profile
 
-The basic building block of a security policy is ComplianceAsCode is a rule. The rule represents a single configuration setting, eg. "Password length is at least 8 characters" or "Logging is enabled".
+The basic building block of a security policy is ComplianceAsCode is a rule.
```

Comment:
Double `is`. Suggestion: The basic building block of a security policy **in** ComplianceAsCode is a rule.